src/routes/measurement/find.ts

```diff
@@ -108,6 +112,10 @@ const controllerGeneratorOptions: ControllerGeneratorOptionsWithClient = {
         hashId: Joi.string().required().example('e13d57'),
         name: Joi.string().required().example('My port'),
       }).required(),
+      edge: Joi.object().keys({
+        hashId: Joi.string().allow(null).example('ka08d'),
+        name: Joi.string().allow(null).example('My line'),
```

Comment:
Shouldn't these two also be marked `.required()`?

Reply:
yes, fixed